src/riak_kv_entropy_manager.erl

```diff
@@ -418,7 +453,12 @@ reload_hashtrees(Ring, State=#state{trees=Trees}) ->
     Existing = dict:from_list(Trees),
     MissingIdx = [Idx || Idx <- Indices,
                          not dict:is_key(Idx, Existing)],
-    _ = [riak_kv_vnode:request_hashtree_pid(Idx) || Idx <- MissingIdx],
```

Comment:
What's the purpose of this change? Seems like the old version was simpler and would have the same effect, unless I'm missing something.